packages/coinjoin/src/client/round/selectRound.ts

```diff
@@ -198,7 +236,7 @@ export const getAccountCandidates = ({
             return groups;
         }, {} as Record<SessionPhase, Parameters<typeof setSessionPhase>[0]>);
 
-        Object.values(eventGroups).forEach(setSessionPhase);
+        Object.values(eventGroups).forEach(group => setSessionPhase(group));
```

Comment:
because foreach loop is using 3 parameters (item, index, array) and setSessionPhase is expecting only one.
i've added a test with check `toHaveBeenLastCalledWith`